places/models.py

```diff
@@ -7,3 +9,20 @@ class Artist(models.Model):
 
```

Comment:
A null=True needs to be added to the website field.

Reply:
Good catch!  Got it!